### ISSUE TITLE
2.9.0 - Cannot read property 'content' of undefined

### ISSUE THREAD

**Issue by gpoitch** [atfzl/eslint-plugin-css-modules#44]
Caused by this new code (https://github.com/atfzl/eslint-plugin-css-modules/pull/42): https://github.com/atfzl/eslint-plugin-css-modules/blob/3b16957800fdfa550dd5e510328e3c6882790395/lib/core/traversalUtils.js#L206

```
TypeError: Cannot read property 'content' of undefined
    at /project/node_modules/eslint-plugin-css-modules/build/core/traversalUtils.js:143:35
    at Node.traverse (/project/node_modules/gonzales-pe/lib/gonzales.js:288:6)
    at Node.traverse (/project/node_modules/gonzales-pe/lib/gonzales.js:293:36)
    at getImportedFilePaths (/project/node_modules/eslint-plugin-css-modules/build/core/traversalUtils.js:138:7)
    at getStyleClasses (/project/node_modules/eslint-plugin-css-modules/build/core/index.js:156:69)
    at /project/node_modules/eslint-plugin-css-modules/build/core/index.js:157:30
    at Array.reduce (<anonymous>)
    at getStyleClasses (/project/node_modules/eslint-plugin-css-modules/build/core/index.js:156:85)
    at ImportDeclaration (/project/node_modules/eslint-plugin-css-modules/build/rules/no-undef-class.js:63:49)
    at /project/node_modules/eslint/lib/util/timing.js:122:13
```

**Comment by atfzl** [comment]
reverting PR #42 in #45

**Comment by atfzl** [comment]
published 2.9.1 after reverting

**Comment by xthule** [comment]
Thanks for the quick fix, @atfzl
